.github/scripts/generate_binary_build_matrix.py

```diff
@@ -381,18 +396,18 @@ def generate_wheels_matrix(
                 continue
 
             if use_split_build and (
-                arch_version not in ["12.4", "12.1", "11.8", "cpu"] or os != "linux"
+                arch_version not in ["12.6", "12.4", "12.1", "11.8", "cpu"] or os != "linux"
             ):
                 raise RuntimeError(
-                    "Split build is only supported on linux with cuda 12.4, 12.1, 11.8, and cpu.\n"
+                    "Split build is only supported on linux with cuda 12.6, 12.4, 12.1, 11.8, and cpu.\n"
                     f"Currently attempting to build on arch version {arch_version} and os {os}.\n"
                     "Please modify the matrix generation to exclude this combination."
                 )
 
             # 12.1 linux wheels require PYTORCH_EXTRA_INSTALL_REQUIREMENTS to install
```

Comment:
This comment seems out of date, isn't it?

Reply:
Thanks for the suggestion, yes, removing 12.1